### PR TITLE
Specter2: Tooltips

### PR DESCRIPTION
**Pull request by lynnerang**
Adds native browser tooltips to all textless icons across the app, e.g.:
- Nav icons
- Missing info flag
- 'Add' and 'remove' icons for annotations, fields, etc

Note: the native tooltips aren't pretty, but after a few days of tinkering with custom versions on the side, Jeff and I ultimately decided to go with these to save time (plus, it's super nice that they can display outside the boundaries of the plugin window).

### CODE REVIEW

**Comment by joeyyang** [comment]
Are you able to include screenshots of these tooltips?

**Comment by lynnerang** [comment]
> Are you able to include screenshots of these tooltips?

Sure, they're just the standard Chrome tooltips that come from providing a `title` attribute:

Screen Shot 2022-04-25 at 11 21 09 AM

File: specter2/src/views/NavBar.svelte [hunk 1]
```diff
@@ -13,6 +14,15 @@
   $: selectedLayers = items ? [...selectedAnnotations, ...items.available] : [];
   $: allSelectionsBounding = items ? selectedLayers.every(({ name }) => name === 'Bounding Box') : false;
 
+  $: getTitle = (itemId) => {
+    let title;
+    if (itemId !== currentView) {
+      const annotationTab = VIEW_NAMES[itemId];
+      title = annotationTab ? `${annotationTab} annotations` : 'Home';
+    }
+    return title;
```
[the remark this answers is not in the document]
Comment (lynnerang) [reply specter2/src/views/NavBar.svelte:23]: either way works, the code here covers all possible cases, so the default value never comes into play

**Review by gblakeman — APPROVED** [review]
🥳🎈

**Review by joeyyang — APPROVED** [review]
Looks good!

**Pull request merged**